src/Illuminate/Routing/Router.php

```diff
@@ -404,7 +404,7 @@ public static function mergeGroup($new, $old)
             $new['as'] = $old['as'].(isset($new['as']) ? $new['as'] : '');
         }
 
-        return array_merge_recursive(array_except($old, ['namespace', 'prefix', 'where', 'as']), $new);
+        return array_merge_recursive(Arr::except($old, ['namespace', 'prefix', 'where', 'as']), $new);
```

Comment:
Using array_* for normal one time operation is fine, however this is going to be hit whenever you use recursive `Route::group()`, I would prefer it it's avoid having to use alias.